test/e2e/v1beta1/scripts/gh-actions/Dockerfile.dummy-collector

```diff
@@ -0,0 +1,9 @@
+FROM python:3.8-slim
+
+WORKDIR /app
+
+COPY test/e2e/v1beta1/scripts/gh-actions/dummy-collector.py .
+
+RUN pip install kubernetes
+
+CMD ["python", "dummy-collector.py"]
```

Comment:
Why do we need this container ?

Reply:
This container is meant for serving as a dummy metric collector container intended for e2e test.